commentHandler.go

```diff
@@ -104,36 +104,72 @@ func findLabel(currentLabels []types.IssueLabel, cmdLabel string) bool {
 	return false
 }
 
+func classifyLabels(currentLabels []types.IssueLabel, labelAction string, labelValue string) ([]string, []string) {
+
+	var goodLabels, badLabels []string
```

Comment:
what is a good/bad label?

Reply:
The good list is those that can have a valid action performed, the bad list is those that cant (so bad would be duplicates in the case of `add` or absentees in the case of `remove`)

Reply:
I don't like the choice of wording - good/bad are too loaded. Please could you try to come up with an alternative suggestion?

Reply:
'Good' and 'Bad' are now 'Applicable' and 'Unnecessary'